Mention Git for Windows CRLF/LF line ending issues (#3891)
godotengine/godot-docs: getting_started/workflow/project_setup/version_control_systems.rst

  project, including sensitive information such as Android keystore credentials.
- ``.mono/``: This folder stores automatically-generated Mono files. It only exists
  in projects that use the Mono version of Godot.

Working with Git on Windows
---------------------------

Most Git for Windows clients are configured with the ``core.autocrlf`` set to ``true``.
This can lead to files unnecessarily being marked as modified by Git due to their line endings being converted automatically.
It is better to set this option as::

>     git config core.autocrlf input

Reviewer: Also, it's probably better to use `--global` to ensure it applies to all repositories:

```suggestion
    git config --global core.autocrlf input
```

Otherwise, it would be equivalent to `--local` which makes it only apply to the repository the CWD is currently in.

Author: Hm not so sure about this, I intentionally left `--global` out. My reasoning is that most clients are configured with `true` and I never had any issue but with Godot. Maybe it's fine and I'm just a worrywart.

  project, including sensitive information such as Android keystore credentials.
- ``.mono/``: This folder stores automatically-generated Mono files. It only exists
  in projects that use the Mono version of Godot.

Working with Git on Windows
---------------------------

> Most Git for Windows clients are configured with the `core.autocrlf` set to `true`.

Reviewer: Also, the syntax for inline code in reStructuredText uses two backticks instead of one.

```suggestion
Most Git for Windows clients are configured with the ``core.autocrlf`` set to ``true``.
```

> Most Git for Windows clients are configured with the `core.autocrlf` set to `true`.
> This can lead to weird git status and warnings in the diff output. It is better to set this
> option as:
> ::

Reviewer: It's better to explain practical consequences rather than being vague :slightly_smiling_face: 

```suggestion
Most Git for Windows clients are configured with the `core.autocrlf` set to `true`.
This can lead to files being marked as modified by Git due to their line endings being converted automatically.
It is better to set this option as::

```

Author: Indeed! Thanks

> 
>   git config core.autocrlf input

Reviewer: ```suggestion
    git config core.autocrlf input
```
